### PR TITLE
CIV-8624 Claimant Intent – Check your Answers Mediation

### PR DESCRIPTION
**Pull request by portizdev**
**Before creating a pull request make sure that:**

- [x] commit messages are meaningful and follow good commit message guidelines
- [x] README and other documentation has been updated / added (if needed)
- [x] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### JIRA link ###

https://tools.hmcts.net/jira/browse/CIV-8624
### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by laarmada — DISMISSED** [review]

File: src/main/routes/features/claimantResponse/claimantResponseTasklistController.ts [hunk 1]
```diff
@@ -23,7 +23,7 @@ claimantResponseTasklistController.get(CLAIMANT_RESPONSE_TASK_LIST_URL, async (r
       description: getDescription(taskLists, lang),
     });
   } catch (error) {
-    next(error);
+    next(error); //
```
Comment (neeta-hmcts) [inline src/main/routes/features/claimantResponse/claimantResponseTasklistController.ts:26]: can be removed

**Review by neeta-hmcts — DISMISSED** [review]
Not related to this PR but In general when defendant rejects the mediation then we should not ask the mediation question to the claimant.. is this already handled ?

File: src/main/modules/i18n/locales/cy.json [hunk 2]
```diff
@@ -481,6 +481,7 @@
       "GIVE_DOC_DETAILS": "Rhoi manylion y dogfennau",
       "GIVE_EVIDENCE": "Ydych chi eisiau rhoi tystiolaeth?",
       "HEARING_REQUIREMENTS_TITLE": "Gofynion ar gyfer gwrandawiad",
+      "YOUR_HEARING_REQUIREMENTS_TITLE": "Eich gofynion ar gyfer gwrandawiad",
```
Comment (laarmada) [inline src/main/modules/i18n/locales/cy.json:484]: please in alphabetic order

File: src/main/routes/features/mediation/freeTelephoneMediationController.ts [hunk 3]
```diff
@@ -11,11 +12,18 @@ freeTelephoneMediationController.get(CITIZEN_FREE_TELEPHONE_MEDIATION_URL, async
   try {
     const civilClaim = await getCaseDataFromStore(generateRedisKey(<AppRequest>req));
     res.render(citizenFreeTelephoneMediationViewPath,
-      {isBusiness: civilClaim?.isBusiness()},
+      {isBusiness: isBusinessUser(civilClaim)},
```
Comment (laarmada) [inline src/main/routes/features/mediation/freeTelephoneMediationController.ts:15]: why use now other different vble name for case_data, please we need to use the same civilClaim...
for example:
image

**Review by kalachandrasekar1 — APPROVED** [review]

**Review by kenneth-hmcts — APPROVED** [review]

File: src/main/services/features/claimantResponse/checkAnswers/buildFreeTelephoneMediationSection.ts [hunk 4]
```diff
@@ -0,0 +1,35 @@
+import { Claim } from 'common/models/claim';
```
Comment (laarmada) [inline src/main/services/features/claimantResponse/checkAnswers/buildFreeTelephoneMediationSection.ts:1]: please have a look the other similar file:
services\features\response\checkAnswers\responseSection\buildFreeTelephoneMediationSection.ts

**Review by neeta-hmcts — APPROVED** [review]

**Review by paul-pearson — APPROVED** [review]

File: src/main/services/features/common/addSupportRequiredList.ts [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+      hearingRequirementsSection.summaryList.rows.push(summaryRow(t('PAGES.CHECK_YOUR_ANSWER.SUPPORT_REQUIRED_DETAILS', {lng}), generateSupportDetails(item, lng), supportRequiredHref, changeLabel(lng)));
+    });
+  } else {
+    hearingRequirementsSection.summaryList.rows.push(
+      summaryRow(t('PAGES.SUPPORT_REQUIRED.TITLE', {lng}), t(`COMMON.${YesNoUpperCase.NO}`, {lng}), supportRequiredHref, changeLabel(lng)),
+    );
+  }
+};
+export const generateSupportDetails = (item: SupportRequired, lng: string) => {
+  const supportDetails = [];
```
Comment (laarmada) [inline src/main/services/features/common/addSupportRequiredList.ts:32]: could we use case instead of a lot of ifs?

**Review by laarmada — DISMISSED** [review]
I need more time for review the rest so sorry

**Pull request merged**